extensions/arc/src/common/utils.ts

```diff
@@ -26,3 +26,12 @@ export function resourceTypeToDisplayName(resourceType: string | undefined): str
 	}
 	return resourceType;
 }
+
+export function parseEndpoint(endpoint?: string): { ip: string, port: string } {
+	endpoint = endpoint ?? '';
+	const separatorIndex = endpoint.indexOf(':');
```

Comment:
we don't need to support commas here, do we (currently, the data controller will always return endpoints with ':' as the delimiter, right?)

Reply:
Also, this is an easy win for a test 😄